internal/feature/flags.go

```diff
@@ -71,6 +72,10 @@ var MainDefaults = Defaults{
 		Enabled: false,
 		Status:  Active,
 	},
+	CustomDeploy: Value{
```

Comment:
what's the point of putting this behind the flag? i feel like features are more for changes in behavior that we don't want to expose yet. we don't usually do them for new tiltfile functions.

Reply:
I need to make a couple breaking changes, specifically new required teardown command, and as part of that I want to switch to exclusively use `cmd` objects from #5143 to avoid having 8 command args.

Given timing w/ release, I didn't want to have a Tilt version with an exposed "beta" `custom_deploy`, so was using this as a way to chunk some of the Tiltfile stuff.

(Happy to also roll those changes into this PR and/or just accept Tiltfile API breakage in this instance)

Reply:
eh, if the flag makes you more comfortable, that's ok - but i don't think the existence of the flag changes anything. (the kinds of people that find an undocumented function will also find the flag that turns it on, and you'll break those people anyway)